bmaptools/BmapHelpers.py

```diff
@@ -83,3 +94,72 @@ def program_is_available(name):
             return True
 
     return False
+
+
+def get_mount_point(filepath):
+    """
+    Return the mount point the file object file resides on.
+    """
+    f = os.path.realpath(filepath)
+    while f != os.path.sep:
+        if os.path.ismount(f): return f
+        f = os.path.realpath(os.path.join(f, os.pardir))
+    return f
+
+
+def get_file_system_type(filepath):
+    """
+    Return the file system type the file object file resides on.
```

Comment:
File object is a python object representing a file, say, something that open() returns. In your case you have just a string (more modern way is to use pathlib.Path and the like, but this codebase uses just strings). Try to improve the comment a bit, see the similar suggestion above.

Reply:
And similar feedback about 'filepath', make it 'path' if this function does not care if it is a file or not. If it cares, may be there needs to be a check?